features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -1198,6 +1198,9 @@
             <Scopes>internal_identity_mgt_create</Scopes>
             <Scopes>internal_identity_mgt_delete</Scopes>
         </Resource>
+        <Resource context="(.*)/api/users/v2/me/webauthn(.*)" secured="true" http-method="all">
+            <Permissions>/permission/admin/manage/identity</Permissions>
```

Comment:
As we access this through the user-portal, isn't the internal_login scope is enough?

Reply:
Since this is a new resource can default to scope instead of permission.

Reply:
Fixed.